lib/api/core/models/security/profile.js

```diff
@@ -71,9 +68,11 @@ class Profile {
       return Bluebird.resolve(false);
     }
 
-    return this.getRoles()
-      .then(roles => new Bluebird((resolve, reject) => {
-        Bluebird.map(roles, role => role.isActionAllowed(request))
+    return this.getPolicies()
+      .then(policies => new Bluebird((resolve, reject) => {
+        Bluebird.map(policies, policy => {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove curly braces and "return" from this arrow function body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3AS3524)